README.md

```diff
@@ -269,46 +269,87 @@ Depending on which robot data you wish to visualize or operation you wish to run
 
 ### Run the Telemetry Server
 
+#### Configure
+
 1. Open a terminal on the same or any other machine connected to the network.
 
 2. If you have used `conda` package manager to install the dependencies as described in Section #server-installation, activate the `conda` environment where you installed these dependencies, otherwise skip to next step.
     ```
     conda activate icubtelemenv
     ```
-
-3. Run `npm start` from `<yarp-openmct-root-folder>/iCubTelemVizServer` folder. You should get on the temrinal standard output something like:
-    ```
-    > iCubTelemVizServer@1.0.0 start <user-home>/dev/yarp-openmct/iCubTelemVizServer
-    > . ${NVM_DIR}/nvm.sh; nvm use v4.2.2; node ${NODE_DEBUG_OPTION} iCubTelemVizServer.js
-    
-    Now using node v4.2.2 (npm v2.14.7)
-    iCub Telemetry server launched!
-    [INFO] |yarp.os.Port| Port /yarpjs/inertial:i active at tcp://192.168.1.100:10117/
-    [INFO] |yarp.os.impl.PortCoreInputUnit| Receiving input from /icubSim/inertial to /yarpjs/inertial:i using tcp
-    ...
-    [INFO] |yarp.os.Port| Port /yarpjs/sysCmdsGenerator/rpc active at tcp://192.168.1.100:10124/
-    OK
-    Opem-MCT static server process started.
-    ICubTelemetry History hosted at http://localhost:8081/history
-    ICubTelemetry Realtime hosted at ws://localhost:8081/realtime
-    listening on http://localhost:3000
-    [OPEN-MCT STATIC SERVER] stdout:
-    > openmctStaticServer@1.0.0 start <user-home>/dev/yarp-openmct/openmctStaticServer
-    > . ${NVM_DIR}/nvm.sh; nvm use v14.17.0; node server.js
-    
-    
-    [OPEN-MCT STATIC SERVER] stdout: Now using node v14.17.0 (npm v6.14.13)
+3. Open the default comfiguration file `<yarp-openmct-root-folder>/config/default.json`. This JSON formatted file holds the ports configuration, the ports prefix and the servers configuration.
```

Comment:
```suggestion
3. Open the default configuration file `<yarp-openmct-root-folder>/config/default.json`. This JSON formatted file holds the ports configuration, the ports prefix and the servers configuration.
```

Reply:
Fixed on my ongoing branch for the next PR.